tests/unit/examples/test_03-Running-on-multiple-GPUs-or-on-CPU.py

```diff
@@ -56,4 +56,9 @@ def test_example_03():
 
             """
         )
+        for cell in tb.cells:
+            cell.source.replace(
+                "device_memory_limit=device_limit", "# device_memory_limit=device_limit"
```

Comment:
why do we need to disable these parameters? and is this related to the change to the dataloader?

Reply:
No this is not related to the dataloader changes.

I think we changed the CI tests. We define a CUDA Cluster with multiple GPUs. We use RMM Pool to reserve GPU memory for this CUDA Cluster. It is more efficient (faster), if we reserve a fixed size of GPU memory. The code tries to reserve X% of total GPU memory (not free). If we run multiple tests in parallel, other tests block GPU memory and we cannot reserve it.

I deactive the parameters that we use the default behavior (not using RMM)

Reply:
maybe worth adding something along these lines as a comment above these lines that may help our future selves recovering context of why these parameters need to be changed